docs/whatsnew/6.0.rst

```diff
@@ -253,6 +255,86 @@ There is no function processing data models, they will be delegated to affiliate
 See :ref:`votable_mivot` for more details.
 
 
+.. _whatsnew-6.0-votable-parquet-serialization:
+
+VOTable now supports PARQUET serialization
+==========================================
+
+The PARQUET file format allows a more efficient handling of large data amounts. However,
+one problem of PARQUET is that it only provides a limited number of column metadata keywords.
+A way to make it consistent with VO standards is to embed it into a VOTable file.
+
+This serialization works similar to the VOTable FITS serialization that already existed. It
+basically creates two files, on VOTable file and one PARQUET file, which are linked together.
+The advantage of this method is that any column metadata can be saved along with the PARQUET file,
+following VO standards.
+
+Reading and writing of the VOTable PARQUET serialization is fully supported by
+`astropy.io.votable` and the unified ``Table`` read/write interface.
+This serialization can be used by setting the ``'format'`` argument to
+``'votable.parquet'``, while ``'votable'`` can be used for reading in such a file.
+The method works for both absolute and relative parquet file paths.
+
+Example for writing:
+
+.. doctest-skip::
```

Comment:
I think if you remove the `>>>`, you do not need `doctest-skip` because doctest won't pick it up, but I guess this also works.

Reply:
This is cleaner, it shows that it's an example and we considered testing it, but decided not to, rather than it wasn't considered. IMO it also looks better.